packages/plugins/GeoLocation/src/store/getters.ts

```diff
@@ -23,6 +23,9 @@ const getters: PolarGetterTree<GeoLocationState, GeoLocationGetters> = {
     }
     return false
   },
+  showTooltip: (_, __, ___, rootGetters): boolean => {
+    return rootGetters?.configuration?.geoLocation?.showTooltip || false
```

Comment:
```suggestion
    return rootGetters.configuration?.geoLocation?.showTooltip || false
```
`configuration` is always defined on `rootGetters`.

Also, wouldn't `Boolean(rootGetters.configuration?.geoLocation?.showTooltip)` suffice?

Reply:
https://github.com/Dataport/polar/pull/12/commits/0fb8ffc0772e3ad117624b51b9ea073b94cbd114

Yep, that's better.